frontend/app/tests/e2e/objects/CoreGraphQLQuery/core-graphql-query.spec.ts

```diff
@@ -17,6 +17,7 @@ test.describe("/objects/CoreGraphQLQuery/:graphqlQueryId - GraphQL Query details
     await test.step("Navigate to CoreGraphQLQuery page", async () => {
       await page.goto("/objects/CoreGraphQLQuery");
       await expect(page.getByRole("heading", { name: "GraphQL Query" })).toBeVisible();
+      await expect(page.getByTestId("object-table-skeleton").first()).toBeHidden();
```

Comment:
Is always true because `<React.Fragment data-testid="object-table-skeleton">` will not render.